doc/user_guide/user_guide.md

```diff
@@ -396,20 +284,33 @@ You can invoke the python script as below which allows to load the transformer
 models from the local filesystem into BucketFS:
 
   ```buildoutcfg
-  python -m exasol_transformers_extension.upload_model \
-      --bucketfs-name <BUCKETFS_NAME> \
-      --bucketfs-host <BUCKETFS_HOST> \
-      --bucketfs-port <BUCKETFS_PORT> \
-      --bucketfs-user <BUCKETFS_USER> \
-      --bucketfs-password <BUCKETFS_PASSWORD> \
-      --bucket <BUCKETFS_NAME> \
-      --path-in-bucket <PATH_IN_BUCKET> \
-      --model-name <MODEL_NAME> \
-      --subd-dir <SUB_DIRECTORY> \
-      --local-model-path <MODEL_PATH>     
+  python -m exasol_transformers_extension.upload_model <options>
   ```
 
-**Note:**  The `--path-in-bucket` can not be empty.
+#### List of options
+
+The table below lists all available options. It shows which ones are applicable for On-Prem and for SaaS backends.
+Unless stated otherwise in the comments column, the option is required for either or both backends.
+
+| Option name                  | On-Prem | SaaS | Comment                                         |
+|:-----------------------------|:-------:|:----:|:------------------------------------------------|
+| bucketfs-name                |   [x]   |      |                                                 |
+| bucketfs-host                |   [x]   |      |                                                 |
+| bucketfs-port                |   [x]   |      |                                                 |
+| bucketfs-user                |   [x]   |      |                                                 |
+| bucketfs-password            |   [x]   |      | Env. [BUCKETFS_PASSWORD]                        |
+| bucketfs-use-https           |   [x]   |      | Optional boolean, defaults to False             |
+| bucket                       |   [x]   |      |                                                 |
+| saas-url                     |         | [x]  |                                                 |
+| saas-account-id              |         | [x]  | Env. [SAAS_ACCOUNT_ID]                          |
+| saas-database-id             |         | [x]  | Optional, Env. [SAAS_DATABASE_ID]               |
+| saas-database-name           |         | [x]  | Optional, provide if the database_id is unknown |
+| saas-token                   |         | [x]  | Env. [SAAS_TOKEN]                               |
+| model-name                   |   [x]   | [x]  |                                                 |
+| path-in-bucket               |   [x]   | [x]  | Root location in the bucket for all models      |
+| sub-dir                      |   [x]   | [x]  | Sub-directory where this model should be stored |
+| [no_]use-ssl-cert-validation |   [x]   | [x]  | Optional boolean, defaults to True              |
```

Comment:
the default should be changed as well at some point

Reply:
Why? We would normally (except when using the Docker-DB) want the validation to be on.

Reply:
ah, I can't read, cert validation is per default on, perfect